oic.r.acl2.raml

```diff
@@ -20,6 +20,24 @@ traits:
   get:
     description: |
       Retrieves the ACL data.
+      When used without query parameters, all the ACE entries are returned.
+      When used with an aceid or resources query parameter, only the
+      ACEs matching the specified parameter are returned.
```

Comment:
Should references to "ACE" in all the descriptions in this file be "ACE2"?

I think it's clear from the context and syntactic definitions what's returned. But that's just me...